sys/log/full/src/log_shell.c

```diff
@@ -99,6 +242,15 @@ shell_log_dump_all_cmd(int argc, char **argv)
             continue;
         }
 
+        if (list_only) {
+            console_printf("%s\n", log->l_name);
+            continue;
+        }
+
+        if ((argc > 1) &&  strcmp(log->l_name, argv[1])) {
```

Comment:
extra space after &&

Reply:
searching for next log could stop after one that was requested was dumped

Reply:
agree, will fix